src/Command/Mute.js

```diff
@@ -0,0 +1,117 @@
+const Command = require('../Command');
+
+module.exports = Command.extend({
+    commandName: 'mute',
+    commandAliases: ['ban'],
+    advertisable: false,
+    processMessage: function (message, tokens) {
+        let hasRoles = message.member !== null && typeof message.member.roles !== 'undefined';
+
+        if (hasRoles && this.memberIsAdministrator(message.member)) {
```

Comment:
Please use early-return instead of deeply nested ifs.  There's a lot of indentation going on here where instead you could do:

```
if (somethingIDontLike) {
    doErrorMessage();
    return; // don't do anything else
}
```